com.unity.robotics.urdf-importer/Runtime/RosSharpDefinitions/Sensor.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml;
+using System.Xml.Linq;
+using System;
+using System.Numerics;
+
+
+namespace RosSharp.Urdf
+{
+    public class Sensor
+    {
+        public string name
+        { get; set; }
+        public string type
+        { get; set; }
+        public string topic
+        { get; set; }
+        private static string k_baseKey = "sensor";
+        private static string k_attributeDelimit = "@";
+        private static string k_elementDelimit = "/";
+        public int updateRate
+        { get; set; }
+        public Dictionary<string,string> elements
+        { get; set; }
```

Comment:
Doesn't will lose the data type of the parsed attribute? I was expecting something like:
`public Dictionary<string,Object> elements { get; set; }`

Reply:
Correction: Just realized xml attributes and element values are  always string. so we need to resolve the type later in the SensorFactory when assigning to the property.